flink-staging/flink-language-binding/flink-python/src/test/python/org/apache/flink/languagebinding/api/python/flink/test/test_custom.py

```diff
@@ -0,0 +1,76 @@
+# ###############################################################################
```

Comment:
Actually, I'm just gonna move this code into the test_main file, I was going to do that in my next test-centric PR anyway.